main.go

```diff
@@ -3,22 +3,23 @@ package main
 import (
 	"context"
 	"fmt"
+	"io/ioutil"
 	"os"
 	"os/exec"
+	"reflect"
+	"sort"
 	"strconv"
 	"strings"
 	"sync"
-	"io/ioutil"
-	"sort"
-	"reflect"
 
 	"github.com/davecgh/go-spew/spew"
+	"github.com/gin-gonic/gin"
 	"github.com/google/go-github/github"
 	"github.com/xanzy/go-gitlab"
 	"golang.org/x/oauth2"
-	"github.com/gin-gonic/gin"
 
 	log "github.com/sirupsen/logrus"
+	"net/http"
```

Comment:
Just curious: what is the criteria for imports order?

Reply:
In general it does std-lib vs 3rd-party. And then sorts them. I see it has misplaced net/http though. curious

Reply:
Exactly :P